planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp

```diff
@@ -39,9 +39,9 @@
 #include <nav_msgs/msg/occupancy_grid.hpp>
 #include <nav_msgs/msg/odometry.hpp>
 #include <tier4_planning_msgs/msg/approval.hpp>
-#include <tier4_planning_msgs/msg/avoidance_factor.hpp>
-#include <tier4_planning_msgs/msg/avoidance_info.hpp>
-#include <tier4_planning_msgs/msg/avoidance_info_array.hpp>
+#include <tier4_planning_msgs/msg/avoidance_debug_factor.hpp>
```

Comment:
@TakaHoribe 
was there any advices that this message needs debug ~ ?
I think previous avoidance factor or avoidance info was better

Reply:
I think this msg can have a debug

Reply:
(might be a little bit late)
I think the message/debug variable is not used anywhere in the code base.
Therefore I believe it is better to have `Debug` added to tell the user (or future contributor) that the variable do not affect any of the `avoidance_module` code.